src/test/java/com/wikia/webdriver/elements/communities/desktop/pages/SpamWikiReviewPage.java

```diff
@@ -0,0 +1,101 @@
+package com.wikia.webdriver.elements.communities.desktop.pages;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Collections;
+import java.util.List;
+
+public class SpamWikiReviewPage extends BasePageObject {
+
+  public enum LANGUAGE_CODE {
+    all, en, ar, de, es, fi, fr, id, hi, hu, it, ja, ka, kn, ko, nl, pt, pl, ru, uk, vi, zh, other
+  }
+
+  public final static String QUESTIONABLE_STATUS = "questionable";
+
+  public final static String QUESTIONABLE_BUTTON_TEXT = "Questionable";
+  public final static String SPAM_BUTTON_TEXT = "Spam";
+  public final static String NOT_SPAM_BUTTON_TEXT = "Not Spam";
+
+  public final static String XPATH_LANGUAGE_COLUMN = "./td[4]";
+  public final static String XPATH_STATUS_COLUMN = "./td[6]";
+
+  public final static String LANG_QUERY_PARAM = "lang=";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)